content/actions/use-cases-and-examples/building-and-testing/building-and-testing-rust.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Building and testing Rust
+intro: You can create a continuous integration (CI) workflow to build and test your Rust project.
+versions:
+  fpt: '*'
+  ghec: '*'
+  ghes: '*'
+type: tutorial
+topics:
+  - CI
+shortTitle: Build & test Rust
+---
+
+{% data reusables.actions.enterprise-github-hosted-runners %}
+
+## Introduction
+
+This guide shows you how to build, test, and publish a Rust package.
+
+{% data variables.product.prodname_dotcom %}-hosted runners have a tools cache with preinstalled software, which includes the dependencies for Rust. For a full list of up-to-date software and the preinstalled versions of Rust, see [AUTOTITLE](/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#preinstalled-software).
+
+## Prerequisites
+
+You should already be familiar with YAML syntax and how it's used with {% data variables.product.prodname_actions %}. For more information, see [AUTOTITLE](/actions/using-workflows/workflow-syntax-for-github-actions).
+
+We recommend that you have a basic understanding of the Rust language. For more information, see [Getting started with Rust](https://www.rust-lang.org/learn).
+
+## Using a Rust workflow template
+
+{% data reusables.actions.workflow-templates-get-started %}
+
+{% data variables.product.prodname_dotcom %} provides a Rust workflow template that should work for most basic Rust projects. The subsequent sections of this guide give examples of how you can customize this workflow template.
+
+{% data reusables.repositories.navigate-to-repo %}
+{% data reusables.repositories.actions-tab %}
+{% data reusables.actions.new-starter-workflow %}
+1. The "Choose a workflow" page shows a selection of recommended workflow templates. Search for "Rust".
+1. Filter the selection of workflows by clicking **Continuous integration**.
+1. On the "Rust - by {% data variables.product.prodname_actions %}" workflow, click **Configure**.
+
+   ![Screenshot of the "Choose a workflow" page. The "Configure" button on the "Rust" workflow is highlighted with an orange outline.](/assets/images/help/actions/starter-workflow-rust.png)
```

Comment:
I'm not sure why this image is shown as broken in the preview. It works fine locally. I'm guessing this is a bug in our new preview system.

Reply:
Asking internally, I found that images are intentionally omitted from the preview. This works fine for me locally